cernopendata/templates/cernopendata_records_ui/terms/detail.html

```diff
@@ -14,7 +14,7 @@ <h2 class="d-inline">
             <div class="row">
               <div class="col-md-12">
                 {% for exp in record.experiment %}
-                <span class="badge badge-success">{{exp.name}}</span>
+                <a href="/search?experiment={{exp.name}}"><span class="badge badge-secondary">{{exp.name}}</span></a>
```

Comment:
Hmm, do we want to remain in the glossary context for these searchable links? Wondering because the other glossary labels work that way... If so, we would have to select also the type glossary for the links. CC @sefeg @daslerr

Reply:
I don't fully understand the question

Reply:
Imagine you are viewing http://opendata-dev.web.cern.ch/glossary/primary and you click on the "trigger" button (which leads to another fellow glossary term). What would you expect when you click on "cms"?  See only CMS-related glossary terms, or see everything we have that is CMS related?  

The current PR  would bring you to: 
- http://opendata-dev.web.cern.ch/search?page=1&size=20&experiment=CMS
representing the latter option.

It could also bring you to:
- http://opendata-dev.web.cern.ch/search?page=1&size=20&experiment=CMS&type=Glossary
representing the former option. (after we fix the query...)

Basically, this is about expected label click preferences... Have you seen any preferences in your user tests?

Reply:
Ah alright, I was confused with the term "link", as I rather see them as "tags". And since they are designed as tags, the expectation should be to see all resources related to it. So, I am inclined to agree to the way the current PR deals with the issue.